backend/src/main/java/com/woowacourse/friendogly/pet/controller/PetController.java

```diff
@@ -28,21 +29,20 @@ public PetController(PetQueryService petQueryService, PetCommandService petComma
     }
 
     @PostMapping
-    public ResponseEntity<Void> savePet(@RequestBody SavePetRequest savePetRequest) {
-        Long petId = petCommandService.savePet(savePetRequest);
-        return ResponseEntity.created(URI.create("/pets/" + petId)).build();
+    public ResponseEntity<SavePetResponse> savePet(@RequestBody @Valid SavePetRequest savePetRequest) {
+        SavePetResponse response = petCommandService.savePet(savePetRequest);
+        return ResponseEntity.created(URI.create("/pets/" + response.id()))
+                .body(response);
     }
 
-    @GetMapping("/{petId}")
-    public ResponseEntity<FindPetResponse> findPet(@PathVariable Long petId) {
-        FindPetResponse findPetResponse = petQueryService.findPet(petId);
-        return ResponseEntity.ok(findPetResponse);
+    @GetMapping("/{id}")
+    public ResponseEntity<FindPetResponse> findById(@PathVariable Long id) {
+        FindPetResponse response = petQueryService.findById(id);
+        return ResponseEntity.ok(response);
```

Comment:
```suggestion
    @GetMapping("/{petId}")
    public ResponseEntity<FindPetResponse> findById(@PathVariable Long petId) {
        FindPetResponse response = petQueryService.findById(petId);
        return ResponseEntity.ok(response);
```

바로 밑에 `memberId`를 쓰는 메서드가 있어서 헷갈릴 것 같은데, 어떠신가요?

Reply:
저는 현재 수준에서는 괜찮아보여요. 추후에 메서드가 많아졌을 때 다시 살펴보도록 하겠습니당!